src/main/java/tools/jackson/databind/datetime/JavaTimeInitializer.java

```diff
@@ -0,0 +1,226 @@
+/*
```

Comment:
Good -- I like this idea -- making it work much like modules, without having to be one.